README.md

```diff
@@ -10,6 +10,8 @@ network.
 
 If cloning the repo directly, you will need to download the C++ API from http://docs.hebi.us/downloads_changelogs.html#software to build the examples and place it in a folder called `hebi-cpp`. If you use the CMake project, it will automatically be downloaded, assuming you have a working internet connection.
 
+For examples which use plotting, matplotlibcpp is needed. Download matplotlibcpp.h from here: https://github.com/lava/matplotlib-cpp and place it in /hebi-cpp-examples/. In terminal, execute "sudo apt-get install python3-matplotlib python3-numpy python3.6-dev"
```

Comment:
Quick note -- we can remove the `matplotlibcpp.h` once we upload our own here, but we should still mention the original library here (and the fact that we changed it) and link to it.

Reply:
Also, do we need `python3.6-dev`, or can we just have `python3-dev` to automatically install the latest?

Reply:
One last thing -- we should add Windows install instructions, and update our Visual Studio project.  I can handle the VS project update.